src/js/BookReader/Mode2Up.js

```diff
@@ -1207,6 +1231,95 @@ export class Mode2Up {
       }
     }
   }
+
+  /**
+   * Calculates and returns leaf spread attributes
+   * + Sets [some] this.br.twoPage properties
+   * @returns  {{
+   *  mainContainerCss: Object,
+   *  spreadCoverCss: Object,
+   *  spineCss: Object,
+   *  leftLeafCss: Object,
+   *  leafEdgeLCss: Object,
+   *  rightLeafCss: Object,
+   *  leafEdgeRCss: Object,
+   * }}
+   */
+  spreadCSSAttributes() {
```

Comment:
_Suggestion_ There's a lot going on in this method and it's is returning a lot of values. I might break this down into a few smaller getters that can build on one another, ie.

```
get baseLeafCss() {
  const top = this.top();
  return {
    position: 'absolute',
    right: '',
    top: `${top}px`,
    zIndex: 2,
  };
}

get leafHeightCss() {
  return {
    height: `${this.br.twoPage.height}px`, // $$$ height forced the same for both pages
  }
}

get leftLeafCss() {
  return {
       ...this.baseLeafCss, <-- pull from the `baseLeafCss` above
       ...this.leafHeightCss,
       left: `${this.br.twoPage.gutter - this.br.twoPage.scaledWL}px`,
       width: `${this.br.twoPage.scaledWL}px`,
  }
}

...

leafCss = this.leftLeafCss;
```